homeassistant/bootstrap.py

```diff
@@ -199,30 +200,32 @@ def from_config_file(config_path, hass=None):
 
     config_dict = config_util.load_config_file(config_path)
 
-    return from_config_dict(config_dict, hass)
+    return from_config_dict(config_dict, hass, args)
 
 
-def enable_logging(hass):
+def enable_logging(hass, verbose=False, daemon=False):
     """ Setup the logging for home assistant. """
-    logging.basicConfig(level=logging.INFO)
-    fmt = ("%(log_color)s%(asctime)s %(levelname)s (%(threadName)s) "
-           "[%(name)s] %(message)s%(reset)s")
-    try:
-        from colorlog import ColoredFormatter
-        logging.getLogger().handlers[0].setFormatter(ColoredFormatter(
-            fmt,
-            datefmt='%y-%m-%d %H:%M:%S',
-            reset=True,
-            log_colors={
-                'DEBUG': 'cyan',
-                'INFO': 'green',
-                'WARNING': 'yellow',
-                'ERROR': 'red',
-                'CRITICAL': 'red',
-            }
-        ))
-    except ImportError:
-        _LOGGER.warn("Colorlog package not found, console coloring disabled")
+    if not daemon:
```

Comment:
We should move this check to the function calling `enable_logging` and have it not call `enable_logging` instead.